modules/ROOT/pages/errors/gql-errors.adoc

```diff
@@ -389,14 +511,61 @@ Status description:: error: syntax error or access rule violation - unsupported
 
 Status description:: error: syntax error or access rule violation - unsupported request. { $input } is not allowed on the system database.
 
+=== 42N19
+
+Status description:: error: syntax error or access rule violation - duplicate clause. Duplicate `{ $syntax }` clause.
+
+=== 42N20
+
+Status description:: error: syntax error or access rule violation - empty list range operator. The list range operator `'[ ]'` cannot be empty.
+
+
+=== 42N21
+
+Status description:: error: syntax error or access rule violation - unaliased return item. Expression in `{ $clause }` must be aliased (use AS).
+
+
+=== 42N22
+
+Status description:: error: syntax error or access rule violation - single return column required. A COLLECT subquery must end with a single return column.
```

Comment:
I don't know. Some of the have qoutes around and some of them just one quote, which would mean that they will render `''OPTIONAL MATCH ...'` or even worse `MATCH ...'`.

Reply:
I think they should be either backticked or without any quotes around them in the code. If they are backticked, then we don't need to do anything in the docs. If they are with single quotes, we have to leave them like this, otherwise, they will appear as `'OPTIONAL MATCH ...'`.

Reply:
But either way, it's a good point, and we should decide how we wanna handle Cypher words.

Reply:
For now, I can add backticks around the ones that are not within single quotes.